packages/gatsby-source-contentful/package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "gatsby-source-contentful",
   "description": "Gatsby source plugin for building websites using the Contentful CMS as a data source",
-  "version": "2.0.67",
+  "version": "3.0.0",
```

Comment:
We are bumping version as part of publishing process, so please revert version bump here. I marked PR with breaking change label to keep track of it

Reply:
@pieh fixed — thanks!